README.md

```diff
@@ -0,0 +1,21 @@
+# Docker Proxy for ghcr.io
+
+This hosts the Dockerfile for building an nginx docker image that will proxy traffic to ghcr.io.
+
+It also caches docker layers, and is RO - no uploads allowed.
+
+To build:
+
+`docker build . -t ghcr.io/opensafely/docker-proxy`
+
+To run
+
+`docker run -d  -e PROXY_DOMAIN=<domain> -e PROXY_ORIGIN=<origin> -e RESOLVER=127.0.01 -e PORT=80 ghcr.io/opensafely/docker-proxy`
```

Comment:
`127.0.0.1`

Reply:
Can you add the documentation for the `<variables>` from `proxy.conf.template` in here, too?

Reply:
I think an example would be useful too. So "given `docker run` above, a `docker pull X` will be reverse proxied to `gchr.io X`" or something like that.

Reply:
Also clarify that the "to run" is more "to try out locally"; and "to deploy" is "git push dokku whatever"?

Reply:
agree, would be good to document e.g. `PROXY_DOMAIN` vs `PROXY_ORIGIN` here

Reply:
Done

Reply:
Hmm, should they be `PROXY_PUBLIC_DOMAIN` and `PROXY_ORIGIN_DOMAIN`?